apps/web/src/hooks/useBlueprint.ts

```diff
@@ -12,10 +12,10 @@ export const useBlueprint = () => {
   const { pathname } = useLocation();
   const segment = useSegment();
   const id = localStorage.getItem('blueprintId');
-  const token = getToken();
+  const { token } = useAuthController();
 
   useEffect(() => {
-    if (id && token !== null) {
+    if (id && !!token) {
```

Comment:
From here on, we'll see several files be updated to pull from the auth controller, this ensures only the auth controller deals with surfacing auth concerns as the source of truth.